SQF/dayz_code/compile/fn_selfActions.sqf

```diff
@@ -120,10 +120,15 @@ if (_inVehicle) then {
 			_temp_keys_names = _totalKeys select 1;	
 			_hasKey = _vehicleOwnerID in _temp_keys;
 			_oldOwner = (_vehicleOwnerID == _uid);
+			
+			{
+				if (_vehicleOwnerID == _x) exitWith {_temp_key_name = _temp_keys_names select _forEachIndex};
+			} forEach _temp_keys;
```

Comment:
Find is simpler to get an index. You can just replace this code below:
```sqf
(_temp_keys_names select (parseNumber _vehicleOwnerID))
```
with this:
```sqf
(_temp_keys_names select (_temp_keys find _vehicleOwnerID))
```
Same for the _characterID one. Then the forEach loops and _temp_key_name variable aren't needed.